sbtc/src/deposits.rs

```diff
@@ -352,9 +352,9 @@ pub struct ReclaimScriptInputs {
     /// script. It is not allowed to exceed the bounds expected for a
     /// 5-byte lock-time in bitcoin-core. It is also not allowed to have
     /// the [`SEQUENCE_LOCKTIME_DISABLE_FLAG`] bit set.
-    lock_time: i64,
+    pub lock_time: i64,
     /// The reclaim script after the <locked-time> OP_CSV part of the script
-    script: ScriptBuf,
+    pub script: ScriptBuf,
```

Comment:
Given we have some validation in the `ReclaimScriptInputs::try_new` fn below, maybe it's better to add some getters and keep the fields private to avoid people from (incorrectly) instantiate this

Reply:
Yeah, I agree.